lib/adiwg/mdtranslator/writers/dcat_us/sections/dcat_us_dcat_us.rb

```diff
@@ -72,7 +72,7 @@ def self.build(intObj, responseObj)
                   json.set!('temporal', temporal)
 
                   json.set!('issued', issued)
-                  # json.set!('accrualPeriodicity', metadataInfo[:metadataMaintenance][:maintenanceFrequency])
+                  json.set!('accrualPeriodicity', metadataInfo[:metadataMaintenance][:frequency])
```

Comment:
Move the implementation into its own file.

It looks like the string value will need to be mapped to the [ISO 8601 values](https://resources.data.gov/schemas/dcat-us/v1.1/iso8601_guidance/#accrualperiodicity). Still take the value from metadataInfo.metadataMaintenance.frequency